aioca/_catools.py

```diff
@@ -656,6 +669,11 @@ async def caget_array(pvs: List[str], **kwargs):
     return results
 
 
+@caget.register(tuple)  # type: ignore
```

Comment:
If we make define 
```python
PVs = Union[Tuple[str, ...], List[str]]

@caget.register(list)  # type: ignore
@caget.register(tuple)  # type: ignore
async def caget_array(pvs: PVs, **kwargs):
```
Then can we get away without `caget_tuple` and the corresponding `@overload`?

Reply:
Good idea, that seems to work.